_posts/2019-10-17-how-to-achieve-cis-benchmark-compliance.adoc

```diff
@@ -842,130 +857,683 @@ If you need to brush up on how the IaC Library works, read the
+  description = "The name of the S3 bucket where cloudtrail logs are delivered."
+  value       = module.root_baseline.cloudtrail_s3_bucket_name
+}
+
+output "cloudtrail_s3_bucket_arn" {
+  description = "The ARN of the S3 bucket where cloudtrail logs are delivered."
+  value       = module.root_baseline.cloudtrail_s3_bucket_arn
+}
+//todo -> same as last
+//output "cloudtrail_kms_key_arn" {
+//  description = "The ARN of the KMS key used by the S3 bucket to encrypt cloudtrail logs."
+//  value       = module.root_baseline.cloudtrail_kms_key_arn
+//}
+//
+//output "cloudtrail_kms_key_alias_name" {
+//  description = "The alias of the KMS key used by the S3 bucket to encrypt cloudtrail logs."
+//  value       = module.root_baseline.cloudtrail_kms_key_alias_name
+//}
+
+output "cloudtrail_iam_role_name" {
+  description = "The name of the IAM role used by the cloudwatch log group."
+  value       = module.root_baseline.cloudtrail_iam_role_name
+}
+
+output "cloudtrail_iam_role_arn" {
+  description = "The ARN of the IAM role used by the cloudwatch log group."
+  value       = module.root_baseline.cloudtrail_iam_role_arn
+}
+----
+
+At this point, you'll want to test your code. See
+link:/guides/foundations/how-to-use-gruntwork-infrastructure-as-code-library#manual_tests_terraform[Manual tests for Terraform code] and
+link:/guides/foundations/how-to-use-gruntwork-infrastructure-as-code-library#automated_tests_terraform[Automated tests for Terraform code]
+for instructions.
+
+Once the module is working the way you want, submit a pull request, get your changes merged into the master branch, and create a new versioned release by using a Git tag. For example, to create a v0.3.0 release:
 
 [source,bash]
 ----
-git tag -a "v0.3.0" -m "Created iam-password-policy module"
+git add landingzone/account-baseline-root
+git commit -m "Add root account baseline wrapper module"
+git tag -a "v0.3.0" -m "Created root account baseline module"
 git push --follow-tags
 ----
 
-With the module released, you can use Terraform to create the password policy. Follow the instructions in the
-link:https://github.com/gruntwork-io/terraform-aws-cis-service-catalog/tree/master/examples/iam-password-policy/terraform[terraform usage example].
-
-[[wrapper_code_terragrunt]]
-===== Using the compliance wrapper module with Terragrunt
+NOTE: This guide will use https://github.com/gruntwork-io/terragrunt[Terragrunt] and its associated file and folder
+structure to deploy Terraform modules. Please note that *Terragrunt is NOT required for using Terraform modules from
+the Gruntwork Infrastructure as Code Library.* Check out link:/guides/foundations/how-to-use-gruntwork-infrastructure-as-code-library[How to use the Gruntwork Infrastructure as Code Library]
+for instructions on alternative options, such as how to
+link:/guides/foundations/how-to-use-gruntwork-infrastructure-as-code-library#deploy_using_plain_terraform[deploying how to use plain terraform].
```

Comment:
is this what we want for this guide too? None of our examples are currently with terragrunt.